docs/time/index.rst

```diff
@@ -54,6 +54,19 @@ To create a |Time| object:
   >>> t[1]
   <Time object: scale='utc' format='isot' value=2010-01-01T00:00:00.000>
 
+
+Note: If the ``times`` array in the above example contains a lot of elements, the ``value`` argument will
+display all the elements of the ``times`` array when the |Time| object ``t`` is called.
+To control the number of elements to be displayed, set the ``threshold`` argument with ``np.printoptions`` as follows:
+
+
+    >>> times = np.arange(1000)
+    >>> t = Time(times, format='cxcsec')
+    >>> with np.printoptions(threshold=10):
+    >>>   print(repr(t))
+    <Time object: scale='tt' format='cxcsec' value=[  0.   1.   2. ... 997. 998. 999.]>
```

Comment:
```suggestion
    ...     print(t.iso)
    <Time object: scale='tt' format='cxcsec' value=[  0.   1.   2. ... 997. 998. 999.]>
    ['1998-01-01 00:00:00.000' '1998-01-01 00:00:01.000'
     '1998-01-01 00:00:02.000' ... '1998-01-01 00:16:37.000'
     '1998-01-01 00:16:38.000' '1998-01-01 00:16:39.000']
```
This is just a suggestion to demonstrate that it also works for more time-like formats, up for discussion.

Reply:
Hi! I see what might have been the problem! This is my first time contributing and making a pull request. Your suggestion is also great. Do I have to send in another pull request or can I just commit suggestions to the file here? Also, what is a changelog and a milestone?